packages/data-service-generator/src/server/app-module/create-app-module.ts

```diff
@@ -37,9 +37,19 @@ const SERVE_STATIC_OPTIONS_SERVICE_ID = builders.identifier(
 );
 const GRAPHQL_MODULE_ID = builders.identifier("GraphQLModule");
 
+type TemplateMappingOptions = {
+  createGraphQLModule: boolean;
+};
+
 export async function createAppModule(
   modulesFiles: ModuleMap
 ): Promise<ModuleMap> {
+  const { appInfo } = DsgContext.getInstance;
```

Comment:
you can write it like that: { appInfo: settings: { serverSettings: { generateGraphQL } }};
instead of having two const